### PR TITLE
new package: iputils 20190709

### PR DESCRIPTION
**Pull request by a-schaefers**
Read: https://k1ss.org/style.txt
Read: https://k1ss.org/guidestones.txt

---

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/iputils/build [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export DESTDIR="$1"
+
+patch -p1 < fix-setuid-redeclared.patch
+
+meson \
+    --prefix=/usr \
+    -DUSE_CAP=true \
+    -DNO_SETCAP_OR_SUID=true \
+    -DUSE_IDN=true \
```
Comment (dylanaraps) [inline community/iputils/build]: `-DUSE_IDN=false` and `libidn` can be removed.

File: community/iputils/build [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    -DBUILD_MANS=false \
+    -Duse_gettext=false \
+    . output
+
+ninja -C output
+
```
Comment (dylanaraps) [inline community/iputils/build]: Remove this line.

File: community/iputils/build [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ninja -C output install
+
+ln -s tracepath "$1/usr/bin/tracepath6"
+ln -s ping "$1/usr/bin/ping6"
```
Comment (dylanaraps) [inline community/iputils/build]: 0210 https://k1ss.org/style.txt

File: community/iputils/depends [hunk 4]
```diff
@@ -0,0 +1,4 @@
+libcap
+libidn2
+meson make
+samurai make
```
Comment (dylanaraps) [inline community/iputils/depends]: 1202  https://k1ss.org/style.txt (Implicit when meson is listed)

File: community/iputils/build [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+patch -p1 < fix-setuid-redeclared.patch
```
Comment (dylanaraps) [inline community/iputils/build]:
```sh
sed -i 's/cap_setuid/cap_setuserid/g'  ninfod/ninfod.c
```

Patch can also be removed.

File: community/iputils/build (hunk 1 above)
Comment (a-schaefers) [reply community/iputils/build]: But then we can't use setcap in post-install?

Comment (dylanaraps) [reply community/iputils/build]: Yes we can. We're removing libidn and not libcap.

File: community/iputils/build [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 export DESTDIR="$1"
 
-patch -p1 < fix-setuid-redeclared.patch
+sed -i 's/cap_setuid/cap_setuserid/g'  ninfod/ninfod.c
```
Comment (dylanaraps) [inline community/iputils/build]: Remove the extra space before `ninfod`.

**Pull request merged**